zipline/modelling/term.py

```diff
@@ -0,0 +1,294 @@
+"""
+Base class for Filters, Factors and Classifiers
+"""
+from numpy import (
+    empty,
+    float64,
+    full,
+    nan,
+)
+
+from zipline.errors import (
+    InputTermNotAtomic,
+    TermInputsNotSpecified,
+    WindowLengthNotPositive,
+    WindowLengthNotSpecified,
+)
+from zipline.utils.lazyval import lazyval
```

Comment:
@twiecki what would you see as the benefit of relative imports here? IMO, relative imports are useful when you expect that any time you move a module, you'd also move the module from which you're importing.  e.g.  If i have
```
spam/
    __init__.py
    foo.py
    bar.py
```
and I expect that `foo.py` and `bar.py` will always be next to each other, then it's useful in `foo` to do something like:
```
from .bar import ...
```
because then if I rename the whole directory, I don't have to change foo.py at all.

Here, on the other hand, I don't think there's any expected relative relationship between `zipline.modelling.term` and `zipline.utils.lazyval`.  The most likely change is that term.py gets moved somewhere else, which would almost certainly break any relative import.